ppa/archive/templates/archive/snippets/pagination.html

```diff
@@ -8,11 +8,11 @@
     <div class="page-controls">
             {# previous link at first (disabled if no previous)  #}
             {% if page_obj.has_previous %}
-            <a rel="prev" class="ui icon basic button" href="?{% querystring_replace page=page_obj.previous_page_number %}">
+            <a name="previous page" title="previous page" rel="prev" class="ui icon basic button" href="?{% querystring_replace page=page_obj.previous_page_number %}">
                 <i class="chevron left icon"></i>
             </a>
             {% else %}
-            <a class="ui disabled icon basic button">
+            <a name="previous page" title="previous page" class="ui disabled icon basic button">
```

Comment:
perhaps disabled should be a `<span>` rather than an `<a>` ? seems misleading to add a title to something that is probably better ignored by a screen reader

Reply:
keeping it as a link allows it to live with the other page links in the screen reader's link table, but still convey the information that it's disabled, which I think is still valuable to the user ("there's normally a link to go to the previous page here, but it's currently disabled") vs no info at all.